packages/react-strict-dom/tests/__snapshots__/html-test.native.js.snap-native

```diff
@@ -1312,57 +1313,27 @@ exports[`<html.*> style polyfills default inherited text styles 1`] = `
         {
           "color": "red",
           "fontFamily": "Arial",
-          "fontSize": 32,
+          "fontSize": 20,
           "fontStyle": "italic",
           "fontVariant": "variant",
           "fontWeight": "300",
-          "lineHeight": 48,
+          "lineHeight": 30,
           "position": "static",
           "textAlign": "right",
           "textTransform": "uppercase",
           "userSelect": "auto",
         }
       }
     >
-      Text should 
-      <Text
-        dir="auto"
-        style={
-          {
-            "color": "red",
-            "fontFamily": "Arial",
-            "fontSize": 64,
-            "fontStyle": "italic",
-            "fontVariant": "variant",
-            "fontWeight": "300",
-            "lineHeight": 96,
-            "position": "static",
-            "textAlign": "right",
-            "textTransform": "uppercase",
-            "userSelect": "auto",
-          }
-        }
-      >
-        inherit
-      </Text>
-       div styles
+      Inherits text styles
     </Text>
     <TextInput
       dir="auto"
-      placeholder="Input should inherit div styles"
+      placeholder="Does not inherit text styles"
       style={
         {
           "borderWidth": 1,
-          "color": "red",
-          "fontFamily": "Arial",
-          "fontSize": 16,
-          "fontStyle": "italic",
-          "fontVariant": "variant",
-          "fontWeight": "300",
-          "lineHeight": 24,
           "position": "static",
-          "textAlign": "right",
-          "textTransform": "uppercase",
```

Comment:
This is a fix to match web, where font styles set on a wrapping div don't get inherited by text inputs